sdk/keyvault/keyvault-secrets/package.json

```diff
@@ -33,10 +33,6 @@
     "README.md",
     "LICENSE"
   ],
-  "browser": {
```

Comment:
What was this for?

Reply:
Adding it back since it's also present in Certificates